bin/polly.py

```diff
@@ -309,19 +314,28 @@ def PositiveInt(string):
     cdir, args.verbosity, args.discard, args.tail, polly_toolchain
 )
 
+if args.cmake:
+  cmake_bin = args.cmake
+else:
+  cmake_bin = 'cmake'
+
+if os.path.isabs(cmake_bin):
+  if not os.path.exists(cmake_bin):
+    sys.exit("CMake binary not found: {}".format(cmake_bin))
+
 if os.name == 'nt':
```

Comment:
if `cmake_bin` is absolute path then it make no sense to run `where`/`which`. Actually `where` will not work. Should be like:
```python
if os.path.isabs(cmake_bin):
  # check exists
else:
  if os.name == 'nt':
    # where
  else:
    # which
```